vignettes/interventions.Rmd

```diff
@@ -0,0 +1,162 @@
+---
+title: "Modelling disease control interventions"
+author: "Sebastian Funk and James M. Azam"
+output:
+  bookdown::html_vignette2:
+    fig_caption: yes
+    code_folding: show
+pkgdown:
+  as_is: true
+bibliography: references.json
+link-citations: true
+vignette: >
+  %\VignetteIndexEntry{Modelling disease control interventions}
+  %\VignetteEncoding{UTF-8}
+  %\VignetteEngine{knitr::rmarkdown}
+editor_options: 
+  chunk_output_type: console
+---
+
+```{r include=FALSE}
+knitr::opts_chunk$set(
+  echo = TRUE,
+  message = FALSE,
+  warning = FALSE,
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+_epichains_ does not provide any direct functionality for studying public health interventions.
+However, the flexible simulation functionality that it includes can be used to consider some specific changes to the parameters that can be interpreted as the result of control measures.
+Here we investigate the effect on outbreak sizes, but the same approaches could be used for investigating chain lengths (using the `statistic` argument to `simulate_summary`) or the time progression of outbreaks (using the `simulate_tree` function).
+
+```{r}
+library("epichains")
+```
+
+As a base case we consider the spread of an infection with a negative binomial offspring distribution with mean 1.2 and overdispersion parameter 0.5.
+We simulate 200 chains tracking up to 99 infections:
+
+```{r simulate_chains}
+sims <- simulate_summary(
+  nchains = 200, offspring_dist = "nbinom", stat_max = 99, mu = 1.2, size = 0.5
+)
+```
+
+We then plot the resulting distribution of chain sizes
+```{r uncontrolled_chains_plot}
+library("ggplot2")
+sims[is.infinite(sims)] <- 100
+ggplot(data.frame(x = sims), aes(x = x)) +
+  geom_histogram(breaks = seq(0, 100, by = 5), closed = "left") +
+  scale_x_continuous(breaks = c(0, 25, 50, 75, 100),
+                     labels = c(0, 25, 50, 75, ">99")) +
+  theme_bw()
+```
+
+# Reductions in the strength of transmission
+
+Following [@lloyd-smith2005] we consider two ways in which disease control interventions can reduce the reproduction number: _population-wide_ and _individual-specific_ control.
+
+## Population-wide control
+
+By population-level control we mean an intervention that reduces the mean number of offspring (i.e. the reproduction number) by a fixed proportion.
+For example, to reduce R by 25% at the population level we scale the `mu` parameter from 1.2 to 0.9:
+
+```{r simulate_chains_pop_control}
+sims <- simulate_summary(
+  nchains = 200, offspring_dist = "nbinom", stat_max = 99, mu = 0.9, size = 0.5
+)
+sims[is.infinite(sims)] <- 100
+ggplot(data.frame(x = sims), aes(x = x)) +
+  geom_histogram(breaks = seq(0, 100, by = 5), closed = "left") +
+  scale_x_continuous(breaks = c(0, 25, 50, 75, 100),
+                     labels = c(0, 25, 50, 75, ">99")) +
+  theme_bw()
+```
+
+## Individual-level control.
```

Comment:
```suggestion
## Individual-level control
```

Reply:
Will keep original to be consistent with population-level sub-heading